_test/test_sqw_file/test_faccess_sqw_v2.m

```diff
@@ -85,10 +84,11 @@
             to = to.init(initob);
             assertEqual(to.npixels,1164180);
 
-            header = to.get_header();
-            assertEqual(header.filename,'slice_n_c_m1_ei140')
-            assertEqual(header.ulabel{4},'E')
-            assertEqual(header.ulabel{3},'Q_\eta')
+            exper = to.get_header();
+            exp_info = exper.expdata;
```

Comment:
currently `get_header` returns `Experiment`. I would rename this method to get_experiment and leave get_header for compatibility, but you, seems were lukewarm to this suggestion. We can live with this naming too. 
as for other naming -- its very confusing indeed, because I am well familiar with the naming, used for similar purposed in Mantid.  Your `IX_experiment` is `exp_info` there so I am constantly mixing the naming